slices/x11-common.yaml

```diff
@@ -0,0 +1,27 @@
+package: x11-common
+
+essential:
+  - x11-common_copyright
+
+slices:
+  data:
```

Comment:
most of these are actually `scripts`. could you please revisit the slice name and whether it should be further split?